Sources/Tests/FeatureTests/pfPasswordStoreTest/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+include_directories(${GTEST_INCLUDE_DIR})
+include_directories(${gtest_SOURCE_DIR}/include ${gtest_SOURCE_DIR})
+include_directories(../../../Plasma/CoreLib)
+include_directories(../../../Plasma/NucleusLib)
+include_directories(../../../Plasma/FeatureLib)
```

Comment:
I know we already have a lot of this pattern, but we should really be trying to clean it up (e.g. using `${CMAKE_SOURCE_DIR}` or `${<PROJECT-NAME>_SOURCE_DIR}` instead of relative paths)...

Reply:
Don't forget [target_include_directories](https://cmake.org/cmake/help/latest/command/target_include_directories.html). Ideally, we'd move to a fully target-based system, but that's beyond the scope of this.

Reply:
I went with `${CMAKE_SOURCE_DIR}` for now. I agree that `target_include_directories` would be great, but we don't _really_ have per-target include directories: they all use the same 4 top-level ones.